Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.6
+
+ENV MICROMDM_VERSION=v1.2.0
+RUN apk --no-cache add curl unzip ca-certificates && \
```

Comment:
Build the dockerfile by cross compiling a linux dependency and using COPY to add the binary to the server, instead of pulling a release. 

Here's a good example of a Dockerfile I've written for another go project with similar startup/configuration:

https://github.com/kolide/fleet/blob/78b831a6d25f9242953b17b7e40b69aee3f698bf/Dockerfile